content/pages/roadmap.html

```diff
@@ -16,61 +16,98 @@ <h4><i class="fa fa-map"></i> <b>Roadmap</b></h2>
+                    <h3>Single Standardized Protocol</h3>
+                    <div>
+                        <p>The idea of moving to a standardized protocol was motivated by the efforts behind the C++ REST API.</p>
 
-                <h3>Multipart Data Objects</h3>
+                        <p>The HTTP protocol is the primary candidate here. It is simple to understand and adding support for it is literally a matter of choosing a good library. S3 has seen major success and it is built entirely on the HTTP protocol. So, it makes a lot of sense to choose HTTP for iRODS. Doing that instantly widens the reach of iRODS.</p>
 
-                <p>As file sizes continue to grow, moving them around monolithically becomes subject to a set of failure modes that are more gracefully handled with the use of smaller files.  iRODS will implement multipart data objects whereby large files are broken up into parts before transfer over the network, and then reconstituted on the other end.</p>
+                        <p>On paper it's a no brainer, however, efforts to move to a protocol such as HTTP will require careful investigation. Performance is a primary factor here. If performance is poor, no one will use iRODS. Not only that, but can everything provided by iRODS today be supported over a standardized protocol such as HTTP?</p>
 
-                <p>Each part of the file will be transferred and tracked as a regular data object within the iRODS protocol.  Any individual part that does not transfer correctly can be retransmitted.  This will allow iRODS to more gracefully overcome network instability, and affords cleaner restarts of large files. This work will also improve transport speeds by eventually serving as the backbone for multi-target transfers into iRODS and multi-source transfers out of iRODS.</p>
+                        <p>The outcome of the C++ REST API will play a huge part in answering that question.</p>
+                    </div>
 
-                <p>Because each part of a larger file will now be addressable, iRODS will be able to provide POSIX-like semantics for object storage technologies.  This means that these object stores can be connected to iRODS in a cache-free configuration.</p>
+                    <h3 id="new-custom-protocol">New Custom Protocol</h3>
+                    <div>
+                        <p>
+                        This approach is about cleaning up the design of the original protocol. From a functionality standpoint, interacting with iRODS would remain nearly identical. The primary difference would be in the design and implementation. The primary benefit of this is that it provides freedom in the sense that it can changed.
```

Comment:
"that it can changed" -> "that it can be changed"

Reply:
done